lib/fluent/plugin/buffer/chunk.rb

```diff
@@ -49,6 +49,7 @@ class Chunk
         def initialize(metadata)
           super()
           @unique_id = generate_unique_id
+          @chunk_id = Fluent::UniqueId.hex(@unique_id)
```

Comment:
This class includes `UniqueId::Mixin`, so we can use `#dump_unique_id_hex` method here.